modules/programs/firefox.nix

```diff
@@ -212,6 +219,10 @@ in {
                   "distribution.searchplugins.defaultLocale" = "en-GB";
                   "general.useragent.locale" = "en-GB";
                   "browser.bookmarks.showMobileBookmarks" = true;
+                  "browser.newtabpage.pinned" = [{
+                    title = "NixOS";
+                    url = "https://nixos.org";
+                  }];
```

Comment:
This is fairly unintuitive behavior. I think that we should at the very least update the description to explain what will happen.

Reply:
That's a great point - I'll update the description!